projects/batfish-common-protocol/src/main/java/org/batfish/datamodel/routing_policy/expr/MatchAsPath.java

```diff
@@ -1,10 +1,12 @@
 package org.batfish.datamodel.routing_policy.expr;
 
 import com.fasterxml.jackson.annotation.JsonCreator;
+import com.google.common.base.MoreObjects;
+import java.util.Objects;
 import org.batfish.datamodel.routing_policy.Environment;
 import org.batfish.datamodel.routing_policy.Result;
 
-public class MatchAsPath extends BooleanExpr {
+public final class MatchAsPath extends BooleanExpr {
```

Comment:
why is it made final ? If that is because it is not (or supposed to be not) extended by anyone, shouldn't NamedAsPath be made final as well?

Reply:
I intend for all of the classes in this package instantiated in the batfish project to be `final`.